make/modules/jdk.accessibility/Launcher.gmk

```diff
@@ -26,77 +26,76 @@
 include LauncherCommon.gmk
 
 ifeq ($(call isTargetOs, windows), true)
-
   ACCESSIBILITY_SRCDIR := $(TOPDIR)/src/jdk.accessibility/windows/native
-  TOOLS_CFLAGS := $(addprefix -I, \
-      $(ACCESSIBILITY_SRCDIR)/include/bridge \
-      $(ACCESSIBILITY_SRCDIR)/common \
-      $(ACCESSIBILITY_SRCDIR)/toolscommon)
 
```

Comment:
It looks like you determined that these aren't all needed in most cases ? Since I don't see them being passed to the various builds any more. jabswitch I'm sure won't need these but I'm a bit more surprised by the others ..

Reply:
We don't need to explicitly set those flags. In for example SetupInspector below, we add `common` and `toolscommon` to EXTRA_SRC, which will implicitly be added to include path, and `include/bridge` is added to EXTRA_HEADER_DIRS, so all of these are accounted for.

Reply:
OK, so EXTRA_SRC implies HEADER search. That's what I needed to know.